prow/external-plugins/needs-rebase/main.go

```diff
@@ -54,6 +54,8 @@ type options struct {
 	updatePeriod time.Duration
 
 	webhookSecretFile string
+
+	cacheValidTime int
```

Comment:
I'd say this should be exposed as a configuration item rather than a CLI option

Reply:
I didn't find any definition of configuration files in this plugin... and it looks to me it make sense to use CLI parameters in this case, along with other parameters.